src/sparseml/modifiers/smoothquant/pytorch.py

```diff
@@ -189,4 +190,5 @@ def _calculate_smoothing_scales(
         scales = activation_scales.pow(self.smoothing_strength) / weight_scales.pow(
             1 - self.smoothing_strength
         )
+        scales = torch.where(weight_scales > 0.0, scales, activation_scales)
```

Comment:
what if weight_scales == 0? why consider the negative case at all if we are using the absolute val

Reply:
i read this backwards.. thanks 😅